### DISCUSSION TITLE
It can be a problem when I used other class names do not include in docs?

### DISCUSSION THREAD

**Discussion by WinKoKoNaing** [nativewind/nativewind#313] [Q&A] [answered]
I found like this in docs 
.....................................................
Compatibility
Class | Native (StyleSheet) | Web (CSS)
-- | -- | --
text-{n} | ✅ | ✅
text-[n] | ✅ | ✅
text-base | ✅ | ✅
......................................................

Other class names(text-xs,text-xl) are working well when I used them in my projects.
I used other classes like this eg. (rounded-xl,rounded-2xl)
Please, I just want to know - is it correct?

**Comment by marklawlor** [comment, marked as answer]
The rounded classes are supported https://www.nativewind.dev/tailwind/borders/border-radius

Be aware that some classes use different scaling systems (eg `w` and `max-w` use different scales). Consult https://tailwindcss.com to ensure your using a valid className.

If a class is not in the docs, please create an issue. I probably just missed it

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
